alfasim_sdk/hook_specs.py

```diff
@@ -1333,6 +1333,52 @@ def calculate_entrained_liquid_fraction(
     """
 
 
+def update_plugins_internal_deposit_layer(ctx: "void*", deposition_layer_thickness: "void*",
+                                             n_control_volumes: "int") -> "int":
+    """
+    **c++ signature** : ``HOOK_UPDATE_PLUGINS_INTERNAL_DEPOSIT_LAYER(void* ctx, void* deposition_layer_thickness,
+    int n_control_volumes)``
+
+    Internal simulator hook to evaluate the thickness of the deposited layer at the inside of the pipeline walls.
+    This is called at the beginning of accounting the diameter reduction.
+
+    The plugin is supposed to change the given ``deposition_layer_thickness`` array pointer. Its values are contiguous
+    in memory and the dimension is given by ``n_control_volumes``. It has unit equal to ``[m]``.
+
+    :param ctx: ALFAsim's plugins context
+    :param deposition_layer_thickness: Thickness of the internal deposition layer
+    :param n_control_volumes: Number of control volumes
+    :returns: Return OK if successful or anything different if failed
+
+    Example of usage:
+    TODO!!!!!!!!
```

Comment:
Better remove the entire block instead of leaving the wrong example in place... this way it is easier to miss the TODO and end up with the wrong example.

Reply:
I think it would be nice to add an example. All documentation has an example. Take a look at them to have some ideas.

Reply:
Agreed. 👍

Reply:
- [ ] Fix example of use